client/e2e/poms/operations.ts

```diff
@@ -25,7 +49,106 @@ export class OperationsPOM {
 
   async urlIsCorrect() {
     const path = this.url;
-    const currentUrl = await this.page.url();
-    await expect(currentUrl.toLowerCase()).toMatch(path.toLowerCase());
+    const currentUrl = this.page.url();
+    expect(currentUrl.toLowerCase()).toMatch(path.toLowerCase());
+  }
+
+  async operationsTableIsVisible() {
+    await this.page.waitForSelector(".MuiDataGrid-root");
+  }
+
+  async operationsViewIsCorrect(role: string, expectedStatuses: string[]) {
+    let statusIndex = 5;
```

Comment:
can this be `statusColumnIndex` instead for clarity? I spent awhile trying to figure out what a status's index was